x-pack/plugins/siem/public/endpoint_alerts/view/details/overview/index.tsx

```diff
@@ -66,6 +71,22 @@ const AlertDetailsOverviewComponent = memo(() => {
     ];
   }, [alertDetailsData]);
 
+  const activeTab = useMemo(
+    () => (alertDetailsTabId ? tabs.find(({ id }) => id === alertDetailsTabId) : tabs[0]),
+    [alertDetailsTabId, tabs]
+  );
+
+  const handleTabClick = (clickedTab: EuiTabbedContentTab): void => {
```

Comment:
:grey_question: Should this be in a useCallback or something?

Reply:
Yep, good catch. Thanks!